### PR TITLE
Add cool.ldr.lfy

### PR DESCRIPTION
**Pull request by yuhldr**
### Please confirm your submission meets all the criteria

- [x] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [x] My pull request follows the instructions at [App Submission][submission].
- [x] I have [built][build] and tested the submission locally.
- [x] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [x] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [x] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub. **Link:**
- [x] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
- [x] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)*

[reqs]: https://docs.flathub.org/docs/for-app-authors/requirements
[maint]: https://docs.flathub.org/docs/for-app-authors/maintanance
[submission]: https://docs.flathub.org/docs/for-app-authors/submission
[build]: https://docs.flatpak.org/en/latest/first-build.html

### CODE REVIEW

File: cool.ldr.lfy.json [hunk 1]
```diff
@@ -0,0 +1,43 @@
+{
+    "app-id": "cool.ldr.lfy",
```
Comment (hfiguiere) [inline cool.ldr.lfy.json:2]: I don't see how `ldr.cool` relate to this.

Is this really under the control of the application developers?

File: cool.ldr.lfy.json [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+        "*.a",
+        "*.pyc"
+    ],
+    "modules": [
+        "pypi-dependencies.json",
+        {
+            "name": "lfy",
+            "builddir": true,
+            "buildsystem": "meson",
+            "build-commands": [],
```
Comment (hfiguiere) [inline cool.ldr.lfy.json:32]:
```suggestion
```

File: cool.ldr.lfy.json (hunk 1 above)
Comment (yuhldr) [reply cool.ldr.lfy.json:2]: sorry, I'm a newcomer, I don't understand what's wrong with `ldr.cool`

Comment (yuhldr) [reply cool.ldr.lfy.json:2]: > I don't see how `ldr.cool` relate to this.
> 
> Is this really under the control of the application developers?

Oh, I understand, the domain name ldr.cool now belongs to me

Comment (bbhtt) [reply cool.ldr.lfy.json:2]: > the domain name ldr.cool now belongs to me

It seems to be for an unrelated android app https://www.coolapk.com/apk/com.lzu.yuh.lzu If it belongs to you please add something about this app on the webpage.

Comment (yuhldr) [reply cool.ldr.lfy.json:2]: > > the domain name ldr.cool now belongs to me
> 
> It seems to be for an unrelated android app https://www.coolapk.com/apk/com.lzu.yuh.lzu If it belongs to you please add something about this app on the webpage.

Can I create a new second-level domain name lfy.ldr.cool or second-level directory ldr.cool/lfy? introduce this software here. The content on the ldr.cool website has other uses at present. Maybe I can introduce this software specifically on ldr.cool in the future.

Comment (bbhtt) [reply cool.ldr.lfy.json:2]: Both works I think.

**Comment by hfiguiere** [comment]
bot, build cool.ldr.lfy

File: cool.ldr.lfy.json (hunk 1 above)
Comment (yuhldr) [reply cool.ldr.lfy.json:2]: > Both works I think.

the website is ok: https://ldr.cool/lfy/

Comment (bbhtt) [reply cool.ldr.lfy.json:2]: Thanks!

**Comment by bbhtt** [comment]
> Build [83531 successful](https://buildbot.flathub.org/#/builders/6/builds/83531) To test this build, install it from the testing repository:
> 
> ```
> flatpak install --user https://dl.flathub.org/build-repo/66165/cool.ldr.lfy.flatpakref
> ```

Seems python-cryptography is missing:

```
python no lib: cryptography, please try: 

 pip install cryptography
python no lib: cryptography, please try: 

 pip install cryptography
```

**Comment by yuhldr** [comment]
> > Build [83531 successful](https://buildbot.flathub.org/#/builders/6/builds/83531) To test this build, install it from the testing repository:
> > ```
> > flatpak install --user https://dl.flathub.org/build-repo/66165/cool.ldr.lfy.flatpakref
> > ```
> 
> Seems python-cryptography is missing:
> 
> ```
> python no lib: cryptography, please try: 
> 
>  pip install cryptography
> python no lib: cryptography, please try: 
> 
>  pip install cryptography
> ```

now, all is ok

File: python3-cryptography.json
Comment (bbhtt) [inline python3-cryptography.json:1]: This should be in the same file as it is autogenerated by pip-generator Run `./flatpak-pip-generator requests cryptography`.

File: python3-cryptography.json [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+    ],
+    "sources": [
+        {
+            "type": "file",
+            "url": "https://files.pythonhosted.org/packages/68/ce/95b0bae7968c65473e1298efb042e10cafc7bafc14d9e4f154008241c91d/cffi-1.16.0.tar.gz",
+            "sha256": "bcb3ef43e58665bbda2fb198698fcae6776483e0c4a631aa5647806c25e02cc0"
+        },
+        {
+            "type": "file",
+            "url": "https://files.pythonhosted.org/packages/85/62/48bcebd955945d8da3fe9b84a679dbf4bf179e1ac36e583b7eaa47506758/cryptography-41.0.5-cp37-abi3-manylinux_2_28_x86_64.whl",
```
Comment (bbhtt) [inline python3-cryptography.json:15]: This is x86_64 only you should either use a source tarball from pypi or add aarch64 binary wheel from pypi and add only-arch

Comment (bbhtt) [reply python3-cryptography.json:15]: A source build requires this:

https://github.com/flathub/io.github.hydrusnetwork.hydrus/blob/ce51152994cd3beb6e240fda31711d6d777d2150/python3-sources.json#L572-L641

https://github.com/flathub/io.github.hydrusnetwork.hydrus/blob/ce51152994cd3beb6e240fda31711d6d777d2150/io.github.hydrusnetwork.hydrus.yml#L6-L7 

and a cargo-sources.json file for cryptography. Extract the tarball from pypi, go into src/rust then use https://github.com/flatpak/flatpak-builder-tools/tree/master/cargo on the Cargo.lock file there.

Comment (yuhldr) [reply python3-cryptography.json]: Cryptography is so cumbersome that this software has removed the cryptography dependency now

File: cool.ldr.lfy.json [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+        {
+            "name": "lfy",
+            "builddir": true,
+            "buildsystem": "meson",
+            "sources": [
+                {
+                    "type": "git",
+                    "url": "https://github.com/yuhldr/lfy",
+                    "tag": "v0.0.3",
+                    "sha256": "9072f40be0d99c529c815bacad35ef624205022033c5d2f8619a690c490f5fc1"
```
Comment (hfiguiere) [inline cool.ldr.lfy.json:37]: sha256 is for files.

Comment (yuhldr) [reply cool.ldr.lfy.json]: ok, new commit

File: cool.ldr.lfy.json [hunk 5]
```diff
@@ -0,0 +1,41 @@
...
+        "python3-requests.json",
+        {
+            "name": "lfy",
+            "builddir": true,
+            "buildsystem": "meson",
+            "sources": [
+                {
+                    "type": "git",
+                    "url": "https://github.com/yuhldr/lfy",
+                    "tag": "v0.0.3"
```
Comment (hfiguiere) [inline cool.ldr.lfy.json:36]: you should still put a commit here in addition to tag.

**Pull request closed without merging**